common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerActivity.java

```diff
@@ -80,6 +87,26 @@ protected void onStop() {
         super.onStop();
     }
 
+    @Override
+    protected void onDestroy() {
+        // If result hasn't been received. It means that this activity is prematurely killed.
+        // In this case, we want to properly cancel the activity so that it doesn't block MSAL from making a subsequent interactive call.
+        if (!mBrokerResultReceived) {
+            final IBrokerResultAdapter resultAdapter = BrokerResultAdapterFactory.getBrokerResultAdapter(SdkType.MSAL);
+            final Bundle resultBundle = resultAdapter.bundleFromBaseException(
+                    new UserCancelException(ErrorStrings.USER_CANCELLED,
+                            "Activity is killed by user"), null);
```

Comment:
Does it require a user to kill it, or can android or the OOM killer take it out randomly?

Reply:
In the IcM, the team does not know who the killer is.

I repro this by killing it myself though. 

Maybe I should change the error code.